clip_text.py

```diff
@@ -30,9 +30,10 @@ def encode(self, docs: Optional[DocumentArray], parameters: dict, **kwargs):
         """
         Encode all docs with text and store the encodings in the embedding attribute of the docs.
         :param docs: documents sent to the encoder. The docs must have text.
-        :param parameters: dictionary to define the `traversal_paths` and the `batch_size`. For example,
-            `parameters={'default_traversal_paths': 'r', 'default_batch_size': 10}` will override the `self.default_traversal_paths` and
-            `self.default_batch_size`.
+        :param parameters: dictionary to define `model_name`, `traversal_paths`, `default_device`, and `batch_size`.
+            For example,
+            `parameters={'model_name': 'ViT-B/16', 'default_traversal_paths': 'r', 'default_device': 'gpu', 'default_batch_size': 10}`
```

Comment:
this is not true. Model cannot be overriden. It is not about overriding parameters, is about setting params specifiic for a single request

Reply:
The docstring meant that default parameters such as model_name with be overriden. Overriding default params is the same as setting params ?

Reply:
This is totally wrong, check the code and tell me what these parameters are doing

Reply:
Thanks for pointing out that I did not actually override some of the default params

Reply:
This is not resolved